### PR TITLE
(#71, maint) Update dependencies for testing

### LINKED ISSUE #71: Vagrant environment does not load ansible.windows modules correctly
### What You Are Seeing?

Specifically in the Vagrant environment, running the tests seems to cause a few errors about `ansible.windows` modules like `win_command` and `win_file` to show up. These don't appear to be problematic in CI, so it may be something with the Vagrant environment specifically or the boxes we're using.

Errors noted include:

```
couldn't resolve module/action 'win_command'. This often indicates a misspelling, missing collection, or incorrect module path

couldn't resolve module/action 'win_file'. This often indicates a misspelling, missing collection, or incorrect module path
```

I tested both with and without fully qualified module syntax (`ansible.windows.<name>`) and both cases failed, which leads me to believe there's something else going on, and it only shows up in Vagrant environment for some reason.

### What is Expected?

Vagrant should run all the tests, the same as the CI runner.

### How Did You Get This To Happen? (Steps to Reproduce)

Run the `build/Invoke-CollectionTests.ps1` script.

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

Update build dependencies.

## Motivation and Context

As I was finalizing #72 I noted that some tests seem to have not been running correctly on the master branch due to missing dependencies. This should resolve that and help ensure our tests are all working correctly.

## Testing

- Running test on Vagrant and in CI

## Change Types Made

* [x] CI/CD fixes
* [ ] Bug fix (non-breaking change)
* [ ] Feature / Enhancement (non-breaking change)
* [ ] Breaking change (fix or feature that could cause existing functionality to change)

## Related Issue

Partially resolves #71 and also addresses some CI issues I found during #72.

## Change Checklist

* [ ] Requires a change to the documentation
* [ ] Documentation has been updated
* [ ] Tests to cover my changes, have been added
* [ ] All new and existing tests passed.
* [ ] PowerShell v2 compatibility checked.

### CODE REVIEW

File: build/Invoke-CollectionTests.ps1 [hunk 1]
```diff
@@ -104,6 +104,17 @@ process {
             $Inventory | Set-Content -Path "~/$InventoryFile" -Force
 
             bash -c $Commands
+
+            $Result = [PSCustomObject]@{
+                Success  = $?
+                ExitCode = $LASTEXITCODE
+            }
+
+            $Result
```
Comment (corbob) [inline build/Invoke-CollectionTests.ps1:113]: I'm not sure if this is going to actually output anything... On my system it's currently erroring on line 143. I changed line 140 to include `Write-Host` and it outputs the hashtable, but without the `Write-Host` there's no output that I see.

**Review by corbob — COMMENTED** [review]
Other than not outputting the result, the code changes look good to me. I'm not able to test is successfully in Vagrant, but we can discuss that tomorrow and go from there.

File: build/Invoke-CollectionTests.ps1 (hunk 1 above)
Comment (vexx32) [reply build/Invoke-CollectionTests.ps1:113]: This branch of code is only executed in CI. Something similar _should_ be happening for Vagrant further down, though. The `throw` is only reached after it's been output... not sure why it wouldn't show.

Comment (vexx32) [reply build/Invoke-CollectionTests.ps1]: Amended the most recent commit to force this to show before the `throw` happens.

**Review by corbob — APPROVED** [review]
LGTM! Thanks for getting this done @vexx32

**Pull request merged**